tests/integration/targets/aci_interface_policy_leaf_fc_policy_group/tasks/main.yml

```
@@ -0,0 +1,231 @@
+      - intf_policy_leaf_access_fc_port_polgrp_present.current[0].infraFcAccPortGrp.children.0.infraRsFcAttEntP.attributes.tDn == 'uni/infra/attentp-policy_group_aep'
+      - intf_policy_leaf_access_fc_port_polgrp_present.current[0].infraFcAccPortGrp.children.1.infraRsFcL2IfPol.attributes.tnFcIfPolName == 'fibre_channel_interface_policy_test'
+      - intf_policy_leaf_access_fc_port_polgrp_present.current[0].infraFcAccPortGrp.attributes.annotation == 'orchestrator:ansible'
+      - intf_policy_leaf_access_fc_port_polgrp_idempotent is not changed
+      - intf_policy_leaf_access_fc_port_polgrp_idempotent.sent == {}
+      - intf_policy_leaf_access_fc_port_polgrp_update is changed
+      - intf_policy_leaf_access_fc_port_polgrp_update.current[0].infraFcAccPortGrp.attributes.descr == 'policygroup description'
+
+  # ADD Leaf Access FC PC Policy Group
+  - name: Adding a interface policy Leaf Access FC PC policy group - check mode works
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group: &fc_pc_present
+      <<: *aci_info 
+      policy_group: fc_pc_test
+      lag_type: port_channel
+      fibre_channel_interface_policy: fibre_channel_interface_policy_test
+      port_channel_policy: port_channel_policy_test
+      attached_entity_profile: policy_group_aep
+      state: present
+    check_mode: yes
+    register: intf_policy_leaf_access_fc_pc_polgrp_check_mode_present
+
+  - name: Adding a interface policy Leaf Access FC PC policy group - creation works
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *fc_pc_present
+    register: intf_policy_leaf_access_fc_pc_polgrp_present
+
+  - name: Adding a interface policy Leaf Access FC PC policy group  - idempotency works
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *fc_pc_present
+    register: intf_policy_leaf_access_fc_pc_polgrp_idempotent
+
+  # UPDATE Leaf Access FC PC Policy Group
+  - name: Adding a interface policy Leaf Access FC PC policy group description - update works
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *fc_pc_present
+      description: policygroup description
+    register: intf_policy_leaf_access_fc_pc_polgrp_update
+
+  - name: Verify present assertions for FC PC Policy Group
+    assert:
+      that:
+      - intf_policy_leaf_access_fc_pc_polgrp_check_mode_present is changed
+      - intf_policy_leaf_access_fc_pc_polgrp_present is changed
+      - intf_policy_leaf_access_fc_pc_polgrp_present.previous == []
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.attributes.name == 'fc_pc_test'
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.children.0.infraRsFcLagPol.attributes.tnLacpLagPolName == 'port_channel_policy_test'
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.children.1.infraRsFcAttEntP.attributes.tDn == 'uni/infra/attentp-policy_group_aep'
```

Assertion check failed

```suggestion
      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.children.1.infraRsFcAttEntP.attributes.tDn == 'uni/infra/attentp-test_aep'
```

---

resolved